### PR TITLE
Georgia Income Tax Rate 2024

### LINKED ISSUE #5186: Georgia Income Tax Rate 2024
The rate is reduced from 5.49% to 5.39% for 2024.

[Reference](https://www.legis.ga.gov/legislation/66260) 
Click current version to see bill text

### LINKED ISSUE #5307: Adjust the labels of the Georgia income tax rate files
Three files currently have "joint" in the label

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #5186
Fixes #5307

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ga/tax/income/main/head_of_household.yaml [hunk 1]
```diff
@@ -11,13 +11,15 @@ metadata:
     # Georgia transitioned to a flat tax rate, starting in 2024
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/head_of_household.yaml:11]: could you adjust the labels while you are in here:
to fix #5307

**Comment by PavelMakarchuk** [comment]
@vrathi101 As far as I read from the [law](https://law.justia.com/codes/georgia/title-48/chapter-7/article-2/section-48-7-20/) Georgia has moved to a flat tax starting in 2024, [these](https://www.house.ga.gov/Documents/CommitteeDocuments/2022/RDC/Tax_Reform_Update.pdf) slides are also helpful 

It should be 5.49% in 2024 and then reduced by 0.1% to 4.99% in 2028 - could you make that adjustment for each bracket and filing status?

**Comment by vrathi101** [comment]
@PavelMakarchuk if 2024 is 5.49% and we reduce by 0.10% per year, then 2028 is gonna be 5.09%. Am I missing something?

**Comment by PavelMakarchuk** [comment]
> @PavelMakarchuk if 2024 is 5.49% and we reduce by 0.10% per year, then 2028 is gonna be 5.09%. Am I missing something?

Yes is should be 4.99% in 2029

**Comment by vrathi101** [comment]
based on this i'm setting 2024 to 5.39% and then decreasing by 0.10% per year until it hits 4.99% - https://www.legis.ga.gov/legislation/66260

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**